spec/bashcov/runner_spec.rb

```diff
@@ -2,7 +2,8 @@
 require 'benchmark'
 
 describe Bashcov::Runner do
-  let(:runner) { Bashcov::Runner.new test_suite }
+  suite = test_suite
```

Comment:
Do you really need that? What about `let(:runner) { Bashcov::Runner.new "bash #{test_suite}" }`?